game/scenes/help_scene.py

```diff
@@ -1,6 +1,10 @@
 import pygame
 from . import SceneBase
 from game.objects import Text, Button, STATE
+from gi.repository import Gdk
+
+SCALE_X = Gdk.Screen.width() / 1200
+SCALE_Y = Gdk.Screen.height() / 900
```

Comment:
SCALE_X and SCALE_Y are being calculated so many times; see if you can calculate them once and get them.

Reply:
Can we import them from a `utils.py` ?  This would still in principle import Gdk, but that would remove the clutter

Reply:
Yes, it could be in another module provided it was executed only once, not once per import.  No, using Gdk would violate modular design.  Gdk should only be used to obtain data for use with Gtk.  Pygame has it's own API for getting the same data.

Reply:
I'm not too familiar with using pygame API to fix the display resolution issue...... I'll look it up and fix this. Thanks for mentioning it :)